runtime/src/iree/hal/drivers/vulkan/registration/driver_module.cc

```diff
@@ -32,6 +32,12 @@ IREE_FLAG(bool, vulkan_tracing, true,
 
 IREE_FLAG(bool, vulkan_robust_buffer_access, false,
           "Enables the Vulkan 'robustBufferAccess' feature.");
+IREE_FLAG(
+    bool, vulkan_sparse_binding, true,
+    "Enables the Vulkan 'sparseBinding' feature (and others) when available.");
+IREE_FLAG(bool, vulkan_sparse_residency, true,
+          "Enables the Vulkan 'sparseResidencyBuffer' feature (and others) "
+          "when available.");
```

Comment:
What does "(and others)" refer to here?

Reply:
other features required (there's a few to get the base functionality)